### PR TITLE
Validate that input files exist before attempting to process them

### LINKED ISSUE #893: [CLI] Unhandled error when specifying an input file that doesn't exist
**Describe the bug**
If I call `squoosh-cli` specifying an input file that doesn't exist, I get an unhandled error.  This is not a huge deal but a small UX issue I think.

**To Reproduce**

Run `squoosh-cli` with an input file that doesn't exist:

```
squoosh-cli --output-dir png --oxipng {} great-moves-keep-it-up.svg
```

**Expected behavior**
I expect the CLI to return a nonzero error code and print a nice console message telling the user that their input file doesn't exist.

**Actual behavior**
Node runtime error:

```
[Error: ENOENT: no such file or directory, open '[FILEPATH]\great-moves-keep-it-up.svg'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: '[FILEPATH]\great-moves-keep-it-up.svg'
}
Worker error:  Error [RuntimeError]: abort(Error: ENOENT: no such file or directory, open '[FILEPATH]\great-moves-keep-it-up.svg'). Build with -s ASSERTIONS=1 for more info.
    at process.w ([REDACTED]\AppData\Roaming\npm\node_modules\@squoosh\cli\build\index.js:7:376640)
    at process.emit (events.js:315:20)
    at processPromiseRejections (internal/process/promises.js:209:33)
    at processTicksAndRejections (internal/process/task_queues.js:98:32)
```

**Version:**
 - OS w/ version: Windows 10
 - Node version: 14.3.0
 - npm version: 6.14.6

### PR DESCRIPTION
**Pull request by fernap3**
This change makes it so that we will only process files that actually exist. For each file that doesn't exist, we'll print a warning to the console. Fixes #893.

### CODE REVIEW

**Comment by fernap3** [comment]
@googlebot I fixed it.

File: cli/src/index.js [hunk 1]
```diff
@@ -182,7 +183,32 @@ function progressTracker(results) {
...
+    }
+
+    validFiles.push(file);
+  }
+
+  return validFiles;
+}
+
 async function processFiles(files) {
+  files = await checkInputFilesValid(files);
```
Comment (fernap3) [inline cli/src/index.js:210]: I'm not sure how people feel about mutating an argument like this. I can refactor if necessary.

**Comment by fernap3** [comment]
New output looks something like this:
image

File: cli/src/index.js (hunk 1 above)
Comment (surma) [reply cli/src/index.js:210]: I think that’s absolutely fine!

**Review by surma — APPROVED** [review]
Cheers! Very much appreciate this!

**Pull request merged**